src/main/java/uk/gov/justice/digital/common/Util.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.digital.common;
+
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.util.Arrays;
+
+public class Util {
+
+    protected void handleError(final Exception e) {
```

Comment:
As far as I can see, this just prints out the exception to stderr. This can be removed and replaced with a call to `logger.error("Some useful message", exception)` wherever this is used.